Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageListView.swift

```diff
@@ -169,6 +168,12 @@ struct MessageListView<Factory: ViewFactory>: View, KeyboardReadable {
         .modifier(HideKeyboardOnTapGesture(shouldAdd: keyboardShown))
     }
     
+    private var shouldShowTypingIndicator: Bool {
+        !channel.currentlyTypingUsersFiltered(currentUserId: chatClient.currentUserId).isEmpty
+            && utils.typingIndicatorPlacement == .bottomOverlay
+            && channel.config.typingEventsEnabled
+    }
+    
```

Comment:
Same suggestion as before: putting it higher at the top of the file. But again, also fine with it being there, just something I'm used to. :)

Reply:
Moved it to the top.